### ISSUE TITLE
Cloud Cost reports in values.yam not generating a report

### ISSUE THREAD

**Issue by ChrisMusgrove** [kubecost/cost-analyzer-helm-chart#2425]
Adding Cloud Cost report to the values.yaml file does not generate a cloud cost report in the ui

**To Reproduce**  

1. Added following to values.yaml:
 assetReports:
    enabled: true # If true, overwrites report parameters set through UI
    reports:
    - title: "Example Asset Report 0"
      window: "7d"
      aggregateBy: "label:squads"
      chartDisplay: "category"
      idle: "separate"
      rate: "cumulative"
      accumulate: false # daily resolution
      # filters:
      #   - property: "cluster"
      #     value: "cluster-one"

 cloudCostReports:
    enabled: false # If true, overwrites report parameters set through UI
    reports:
      - title: "Cloud Cost Report 0"
        window: "today"
        aggregateBy: "type"
        accumulate: false # daily resolution

3. push to our kubecost setup 

4. Asset report generates successfully, no cloud cost report generates

**Expected behavior**  
Cloud Cost report is generated in Kubecost UI

**What impact will this have on your ability to get value out of Kubecost?**
Unable to generate cloud cost reports, with Asset reports being deprecated will be unable to generate required reporting for costs

**Bug report**
[kubecost-bug-report.txt](https://github.com/kubecost/cost-analyzer-helm-chart/files/12000381/kubecost-bug-report.txt)

**Please share the support case, if any**  
4533

**Comment by kirbsauce** [comment]
@kwombach12 , diagnosis by support team feels this needs reviewed by engineering.

**Comment by kwombach12** [comment]
Thanks @kirbsauce. We will take a look!

**Comment by avrodrigues5** [comment]
Hello @ChrisMusgrove ,

Please give cloudCostReport in values.yaml as below

``` 
    cloudCostReports:
      enabled: true # If true, overwrites report parameters set through UI
      reports:
        - title: "Cloud Cost Report 0"
          window: "7d"
          aggregateBy: "service"
          # filters: 
          #   - property: "service"
          #     value: "service-as-seen-in-Cloud Cost UI" 
```

Filter is optional but you are more than welcome to try it. 

https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/saved-reports#managing-reports-via-values.yaml, Per the doc the enabled should be set to true and I can confirm assetReport block that you pasted if enabled is false there it wont create a asset Report as well. 

Also default aggregate by in the values.yaml was set to "Type" its not a valid aggregate by option for cloud cost ... I have a [PR](https://github.com/kubecost/cost-analyzer-helm-chart/pull/2433) to update it to service so that there isnt any confusion. 

LMK if there's anything else i can help with!

CC: @cliffcolvin

**Comment by ChrisMusgrove** [comment]
Thanks for that, that works a treat! 

If we want to seperate costs by label could you please let us know the format of the aggregateBy, our attempts have yet to work.

**Comment by ChrisMusgrove** [comment]
We are still having an issue:

 assetReports:
    enabled: true # If true, overwrites report parameters set through UI
    reports:
    - title: "Example Asset Report 0"
      window: "7d"
      aggregateBy: "label:squads"
      chartDisplay: "category"
      idle: "separate"
      rate: "cumulative"
      accumulate: false # daily resolution
      # filters:
      #   - property: "cluster"
      #     value: "cluster-one"

  cloudCostReports:
    enabled: true # If true, overwrites report parameters set through UI
    reports:
    - title: "Cloud Cost Report 3"
      window: "7d"
      aggregateBy: "label:squads"

Asset report builds, however the same aggregate by in a cloud report fails, even though this can be done in the ui (although cannot be saved, another bug we raised)
all other cloud reports we tested work fine, just ones by labels that do not

**Comment by avrodrigues5** [comment]
@ChrisMusgrove , Yes there is a bug related to labels while creating cloud cost reports through helm. I'll triage and get back. 

CC: @cliffcolvin

**Comment by avrodrigues5** [comment]
@ChrisMusgrove , the fix should be in 1.105 when it gets released!

**Comment by ChrisMusgrove** [comment]
@avrodrigues5 looks to be working now in v1.105 thanks
